primer/src/Primer/Eval/Redex.hs

```diff
@@ -794,6 +795,7 @@ viewLets e = do
       LetType m a ty b -> Just ((m, LetTyBind (LetTypeBind a ty)), b)
       _ -> Nothing
 
+-- TODO: rename?
 unviewLets :: [(ExprMeta, LetBinding)] -> Expr -> Expr
```

Comment:
Continuing the last bikeshedding thought: I'm not particularly happy with this name. Currently it is ok as it is the inverse of `viewLets`, but later we tweak `viewLets` so it is not. Elsewhere we have `{un,fold}TApp` as a similar view (though that takes a monadic argument to fill in metas). Perhaps keep `viewLets` (as it will be tweaked later` and change this to `foldLetBindings`?

Any better ideas?

Reply:
`foldLetBindings` sounds better. I don't have any other ideas.